### PR TITLE
Feature : add a page for program committee

### PR DESCRIPTION
**Pull request by nicmarti**
Create a new web page to show the program committee members.

For Devoxx FR, some of the webuser are just "admin" and are not active members of the review program. As such, I addded a "Webuser.isPublicVisible(uuid:String)" method. If returns true then show the profile on the web page, else hide.

### CODE REVIEW

File: app/views/CFPAdmin/showCFPUsers.scala.html [hunk 1]
```diff
@@ -27,31 +27,34 @@ <h3><i class="fas fa-user-md"></i> Configure track leader</h3>
...
                             </td>
                         </tr>
-                        }
+
+                    }
                 </tbody>
             </table>
-<button value="save" type="submit" class="btn btn-success"><i class="fas fa-save"></i> Save the full list</button><br>
-<a href="@routes.Backoffice.homeBackoffice()" class="btn btn-primary">Go back to Backoffice</a>
+            <button value="save" type="submit" class="btn btn-success"><i class="fas fa-save"></i> Save the full list</button><br>
```
Comment (aheritier) [inline app/views/CFPAdmin/showCFPUsers.scala.html]: The form and the button are useless ?

**Review by aheritier — APPROVED** [review]
LGTM @nicmarti 
Just a remark on the "Save the full list" button which is useless I think
The test is ok on my side

File: app/views/CFPAdmin/showCFPUsers.scala.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@(allTrackLeaders:Seq[(String, String)])(implicit lang: Lang, flash: Flash, req:RequestHeader)
+@(allCFPWebusers:List[Webuser])(implicit lang: Lang, flash: Flash, req:RequestHeader)
 
 @import helper.twitterBootstrap._
 @import org.apache.commons.lang3.StringUtils
```
Comment (aheritier) [inline app/views/CFPAdmin/showCFPUsers.scala.html:4]: line 6, the page title is wrong

File: app/views/CFPAdmin/showCFPUsers.scala.html (hunk 1 above)
Comment (nicmarti) [reply app/views/CFPAdmin/showCFPUsers.scala.html]: Indeed. I removed the code 
The @form also was not required. thanks

File: app/views/CFPAdmin/showCFPUsers.scala.html (hunk 2 above)
Comment (nicmarti) [reply app/views/CFPAdmin/showCFPUsers.scala.html:4]: Fixed. 
Thanks :-)

**Pull request merged**